wiki/en/Tips-Tricks-More.md

```diff
@@ -80,21 +80,92 @@ Here is the script:
+When this option is used on the command line, Jamulus will display a server channel number assigned to each client, which can be used to control the channel using MIDI CC numbers.  (Note: This is only available for use with macOS and Linux using Jamulus version 3.7.0 or higher, and on Windows using the Jamulus version with JACK support.  It also relies on the Jamulus UI being displayed.)
 
-`--ctrlmidich "1;f0*8;p16*8;s32*8;m48*8;o64"`
+Make sure you connect your MIDI device's output port to the Jamulus MIDI in port (QjackCtl (Linux/Windows), MIDI Studio (macOS) or whatever you use for managing connections). In Linux you will need to install and launch a2jmidid so your device shows up in the MIDI tab in Qjackctl.
 
-Here, Jamulus listens on MIDI channel 1. Volume fader CC numbers start at 0 and there are 8 of them (so end at CC number 7). Pan controls start at CC number 16 and end at 23; Solo 32 to 39 and Mute 48 to 55. Mute Myself is enabled/disabled by CC number 64.
+`--ctrlmidich` takes a single argument.  If you omit it, the parameter is ignored.  There are two formats for the argument:
 
-Please note that for the functions controlled by buttons to work properly, your MIDI controller needs the buttons to be set to "toggle" mode. This means that when pressed to 'turn on' a control, it must send a MIDI CC number with a value >=64, and to 'turn off' the control it must send the same CC number with a value <64. You can read your controller's manual to find out how to set this.
+1. The legacy definition has one or two numbers in the format:
 
-*Note*: Jamulus does not provide feedback on the on/off state of buttons, meaning that your controller must keep track and toggle LEDs (if any) to 'on' or 'off' itself.
+   ```
+   [MIDI channel];[offset for first fader]
+   ```
 
-Fader strips in the mixer window are controlled in ascending order from left to right. Continuing with the above example, in strip number 1 (farthest left), the volume fader would be controlled by CC number 0; pan by 16; solo by 32 and mute by 48. As we have specified 8 consecutive controllers for each parameter, this would give us MIDI control over 8 strips (volume, pan, solo and mute in each one) in the mixer window. The next strip would be controlled by 1, 17, 33 and 49, and so forth.
+   * `MIDI channel` defaults to `0`, meaning "any channel"; value may be omitted if ";..." is present.
 
-Make sure you connect your MIDI device's output port to the Jamulus MIDI in port (QjackCtl (Linux/Windows), MIDI Studio (macOS) or whatever you use for managing connections). In Linux you will need to install and launch a2jmidid so your device shows up in the MIDI tab in Qjackctl.
+   * `offset for first fader` is the first MIDI CC to use to control a Jamulus Channel fader (default 70 for the sake of Behringer X-Touch), with all MIDI CCs after that being used; must be a number to use the legacy format.
```

Comment:
Not sure what's meant by "for the sake of Behringer X-Touch") here.

Reply:
Original author's prerogative, I think.

Reply:
Does it mean "in the case of the Behringer X-Touch"? If so I'd change it to that.

Reply:
No, it doesn't.  It means for the sake of the Behringer X-Touch, which is what the original author used.  "In the case of" would mean there was some knowledge of what was connected and there isn't.

Reply:
OK, so what does "for the sake of Behringer X-Touch" mean exactly because I can't parse that. Is it "to be compatible with Behringer X-Touch"?

Reply:
Updated.  Plus also corrected something I'd not understood in the parsing.